client/src/pages/app/mainApp.wrapper.tsx

```diff
@@ -11,11 +11,34 @@ import RentalRequestsPage from './rental-requests.page';
 import BrowseUserCars from './userCarsPage';
 import BrowseUserRentalRequests from './rentalrequestPage';
 import UpdateCarForm from '../forms/updateCar.form';
+import { getUserIdFromToken } from '../../services/account.service';
+import { toast } from 'react-toastify';
```

Comment:
Wouldn't it be better if we configured notifications toasters in axios helper as well, then only call axios helper here?

Reply:
no, the communication mechanism isn't based on HTTP requests so axios cannot be used